README.md

```diff
@@ -106,6 +106,53 @@ JsonUtils::decodeObject($json); // decodes a JSON string, use stdClass to repres
 JsonUtils::decodeArray($json); // decodes a JSON string, use arrays to represent JSON objects (same as json_decode($value, true))
 ```
 
+### Log\LogUtils
+
+This class provides a useful truncate method that truncates long strings for logging purposes.
+
+``` php
+<?php
+
+LogUtils::truncates('error', 3); // returns "err"
+```
+
```

Comment:
There is a typo on the method name :)

The intended usage is to call it without a second argument, like this:

```
LogUtils::truncate($response->getBody())
```

This prevents logging accidentally large strings.

Could you add that the default length is 255, and that we can disable truncating by setting the `NO_TRUNCATE_LOG` environment variable to `1` ?